doc/source/user/ionumpy.rst

```diff
@@ -0,0 +1,20 @@
+================================================
+How to read and write data using NumPy
+================================================
+
+.. currentmodule:: numpy
+
+.. testsetup::
+
+   import numpy as np
+   np.random.seed(1)
+
+**Objectives**
+
+- Writing NumPy arrays to files
+- Reading NumPy arrays from files
+- Dealing with encoding and dtype issues
+
+**Content**
+
+To be completed.
```

Comment:
@melissawm I think you just added one so you could put in a `toctree`? In that case this seems fine, and this document can be written soon. It'd be best not to put more placeholders in though imho, better to use issues for that.

Reply:
That's right, I just wanted it to be in the toctree. I agree about no more placeholders .